src/styles/index.module.css

```diff
@@ -0,0 +1,63 @@
+.container {
+    padding-top: 100px;
+    height: 90vh;
```

Comment:
remove this height constraint, it doesn't scale well with more items.

You end up with something like this when it overflows 90% of viewport height
![Screen Shot 2021-09-24 at 12 15 21 PM](https://user-images.githubusercontent.com/11952174/134707716-c30ff589-a423-4213-8d63-3e431430cc44.png)